examples/raspberrypi/rp2xxx/src/cyw43.zig

```diff
@@ -0,0 +1,43 @@
+//! This example is work in progress on CYW43xx WiFi/BT driver
+//! Tested on Pico W and Pico 2 W
+//! Driver code based on: https://github.com/embassy-rs/embassy/tree/main/cyw43
+const std = @import("std");
+const microzig = @import("microzig");
+
+const rp2xxx = microzig.hal;
+const time = rp2xxx.time;
+const gpio = rp2xxx.gpio;
+
+const drivers = microzig.hal.drivers;
+const CYW43_Pio_Device = drivers.CYW43_Pio_Device;
+
+const uart = rp2xxx.uart.instance.num(0);
+const baud_rate = 115200;
+const uart_tx_pin = gpio.num(0);
+
+pub fn panic(message: []const u8, _: ?*std.builtin.StackTrace, _: ?usize) noreturn {
+    std.log.err("panic: {s}", .{message});
+    @breakpoint();
+    while (true) {}
+}
+
+pub const microzig_options = microzig.Options{
+    .log_level = .debug,
+    .logFn = rp2xxx.uart.logFn,
+};
+
+pub fn main() !void {
+    uart_tx_pin.set_function(.uart);
+
+    uart.apply(.{
+        .baud_rate = baud_rate,
+        .clock_config = rp2xxx.clock_config,
+    });
+
+    rp2xxx.uart.init_logger(uart);
+
+    var cyw43: CYW43_Pio_Device = .{};
+    try cyw43.init(rp2xxx.pio.num(0), rp2xxx.gpio.num(25), rp2xxx.gpio.num(24), rp2xxx.gpio.num(29), rp2xxx.gpio.num(23));
```

Comment:
you import gpio so you don't need `rp2xxx.`. Could do the same for pio (if you want)